libminifi/include/utils/GeneralUtils.h

```diff
@@ -61,6 +61,17 @@ T exchange(T& obj, U&& new_value) {
 template<typename...>
 using void_t = void;
 
+template<class T>
+using remove_reference_t = typename std::remove_reference<T>::type;
+
+template<class T>
+using remove_cv_t = typename std::remove_cv<T>::type;
```

Comment:
Please add a fallback to std to the `#else` branch of this `#ifdef` block.

Reply:
Updated as requested.